tests/devapps/DesktopTestApp/PublicClientHandler.cs

```diff
@@ -40,11 +40,7 @@ class PublicClientHandler
         public PublicClientHandler(string clientId)
         {
             ApplicationId = clientId;
-            PublicClientApplication = PublicClientApplicationBuilder.Create(ApplicationId)
-                .WithComponent(_component)
-                .BuildConcrete();
-
-            PublicClientApplication.UserTokenCache.Deserialize(TokenCacheHelper.GetUserCache().Serialize());
+            CreateOrUpdatePublicClientApp(InteractiveAuthority, ApplicationId);
```

Comment:
I updated this sample app as it was throwing null refs because of the token cache. 

@MarkZuber  - line 47, which is a still using valid MSAL public API, leads to the Null Ref exception I was talking about in my email.